xlators/mgmt/glusterd/src/glusterd-geo-rep.c

```diff
@@ -699,10 +683,7 @@ glusterd_query_extutil_generic(char *resbuf, size_t blen, runner_t *runner,
                                            void *data))
 {
     int ret = 0;
-    xlator_t *this = NULL;
-
-    this = THIS;
-    GF_ASSERT(this);
+    xlator_t *this = THIS;
```

Comment:
nit: 'this' is only used in this function in error messages, and then they bail out. You could have replaced each 'this->name' with 'THIS->name' in this function and avoid an unconditional call to 'THIS' here.

For a follow up patch series, not in this one of course...

Reply:
Ack. Adding it in the issue as comment.